packages/ag-charts-enterprise/src/features/color-picker/colorPickerStyles.css

```diff
@@ -48,6 +48,12 @@
     filter: drop-shadow(0 0 5px #00000038);
 }
 
+.ag-charts-color-picker__palette:active,
+.ag-charts-color-picker__palette:focus {
+    outline: 1px solid var(--ag-charts-active-color);
```

Comment:
It feels like the right value to use here would be `outline: var(--ag-charts-focus-border);` so that we're consistent with the focus indicator used for scene nodes:

https://github.com/ag-grid/ag-charts/blob/3b828781e57c8ae9e424118d8c92a7c784ca263b/packages/ag-charts-community/src/chart/dom/focusStyles.css#L12

We may even want to consider adding a shared CSS focus-box class for the color-pick and scene-graph focus-indicator